src/settings/settings.ts

```diff
@@ -69,6 +71,17 @@ export class TASettingsTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     }));
 
+        new Setting(containerEl)
+            .setName('Add space after Autocomplete')
+            .setDesc('Enable/disable adding space at the end of the autocompleted word.')
```

Comment:
This makes more sense to me.
```suggestion
            .setName('Space terminator after autocomplete')
            .setDesc('Enable/disable adding space terminator to autocompleted words.')
```